tool/extract.dart

```diff
@@ -112,5 +111,5 @@ void clean() {
   Iterable<FileSystemEntity> files = exampleDir
       .listSync()
       .where((FileSystemEntity entity) => entity is File && entity.path.endsWith('.dart'));
-  files.forEach((File file) => file.deleteSync());
+  files.forEach((FileSystemEntity file) => file.deleteSync());
```

Comment:
Why is this needed given the where clause above filters to files only?

Reply:
This was to fix a strong mode issue, but I updated it to make the definition of `files` more specific (`Iterable<File>`).